### ISSUE TITLE
In case of error do not return dataframe

### ISSUE THREAD

**Issue by grafa** [dashee87/cryptory#1]
Hi there
just a small suggestion. Once an error occured I wouldn't return a DataFrame with error but more like throw custom exception or return False/None.

It's much easier to catch that instead of look into the DataFrame.
In other words once I get DataFrame I want to have data.

Thank you.

**Comment by dashee87** [comment]
Yeah, that's fair. I can't really remember why I went with dataframes for errors, but I blame my R background. I'll replace those dataframes with more conventional exceptions. Thanks for bringing this up!

**Comment by grafa** [comment]
dont worry man :wink:

**Comment by dashee87** [comment]
Ah, not really. I haven't had much time for the last few months to work on these side projects. But I should have a bit more time in the coming weeks.

**Comment by dashee87** [comment]
Finally... the package now raises a generic exception instead of returning a dataframe.  I'll close this issue now. 

It would be nice to catch some specific exceptions (e.g. timeout, no internet, etc.), so I'll set myself the goal of doing that by 2023. Thanks for highlighting this issue!
